98 generic page template

sgnl opened the pull request:
resolves #98

JoeKarlsson commented:
Nice job, it looks great! Add the contact us button on the bottom of the Privacy page, and also add the horizontal lines to both the faq and the privacy page.

sgnl commented:
- added the horizontal lines via CMS
- added email reveal button to the bottom of the privacy page via CMS
- updated reveal button code: using dumb obfuscation techinique for email address

JoeKarlsson commented:
LGTM! :white_check_mark:
